back/api/locale/fr/LC_MESSAGES/django.po

```diff
@@ -589,7 +589,11 @@ msgstr "Geoshop - Devis demandé"
 
 #: api/models.py:775
 msgid "A new quote has been requested:"
-msgstr "Un devis a été demandé:"
+msgstr ""
+"Un devis a été demandé. Il s'agit soit d'un produit dont la tarification est "
+"manuelle, soit d'une commande utilisateur permanent dont le client ou le "
+"contact n'est pas encore définit comme tel dans l'admin. Il faut contrôler "
```

Comment:
```suggestion
"contact n'est pas encore défini comme tel dans l'admin. Il faut contrôler "
```

Reply:
I really like this translation ;-)

It looks like there is quite more to say in French....

(I am laughing...)